Variation cleanup, improved debug colors

dimgrav opened the pull request:
Looks like Pop Light is basically Pop Color, but with a few malconfigured colors, as seen below:

![Screenshot from 2020-09-28 20-59-00](https://user-images.githubusercontent.com/24510850/94468968-ecefe400-01cd-11eb-9b25-5902238f2b20.png)

I merged Pop Light with Pop Color and left two valid variations (light/dark). Also provided some consistent debug color styles for both variations (issue #8 for reference).

dimgrav committed:
Merge problematic Pop Light with Pop Color, keep two variations

dimgrav committed:
Improve visibility of Pop Dark debug colors

subpop commented on poptheme/themes/Pop-Light-color-theme.json:
```diff
@@ -1,207 +1,207 @@
-         "dropdown.background": "#F6F6F6",
-         "dropdown.foreground": "#403C3A",
-         "dropdown.border": "#63524b00",
+		"dropdown.background": "#F6F6F6",
```
Looks like you've got some tab vs. space issues here. I'd suggest cleaning these up to make a smaller diff that contains *just* the changes you're proposing in this PR.

dimgrav replied:
Ah, didn't notice that, thanks for pointing it out!

ArtisanByteCrafter replied:
Agreed, once PR is only proposed changes, i'm happy to review and merge.

ArtisanByteCrafter commented:
Thanks for the contributions! I haven't had much time to work on this lately, but I'll be sure to review soon and get it updated once we get the issues raised all sorted out.

ArtisanByteCrafter reviewed (changes requested):
The changes i think you meant look good to me. If you could sort the indent issues in the diff, it would be easier to review, then I'll be happy to merge this PR. Thanks!

dimgrav commented:
@ArtisanByteCrafter will do the necessary cleanup asap, unfortunately I didn't have time this week!

dimgrav committed:
Resolve whitespace issues

dimgrav commented:
@ArtisanByteCrafter  I deleted the original `Pop-Light-color-theme.json` and renamed `Pop-color-theme.json` -> `Pop-Light-color-theme.json`, then re-applied the specific changes per line.

dimgrav committed:
Merge upstream/master in master

subpop commented:
@dimgrav take a look a the resulting patch set produced by the PR https://github.com/ArtisanByteCrafter/VSCodePopTheme/pull/11/files. It still contains changes to whitespace only. Is this a tabs vs. spaces issue perhaps?

dimgrav commented:
> @dimgrav take a look a the resulting patch set produced by the PR https://github.com/ArtisanByteCrafter/VSCodePopTheme/pull/11/files. It still contains changes to whitespace only. Is this a tabs vs. spaces issue perhaps?

The whitespace diff is due to renaming `Pop-color-theme.json` -> `Pop-Light-color-theme.json`. They don't share the same indentation.

subpop commented:
Ahhh yea, that makes sense.

ArtisanByteCrafter commented:
Version 1.0.4 has been published to the VSCode Marketplace to resolve these issues. Thanks again!
